Emby.Server.Implementations/Library/Resolvers/PlaylistResolver.cs

```diff
@@ -1,4 +1,5 @@
 #pragma warning disable CS1591
+#pragma warning disable SA1600
```

Comment:
what is this warning btw?

Reply:
The same at the one above it, but from the stylecop analyzer
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1600.md